.github/workflows/slack_notify_opened.yml

```diff
@@ -0,0 +1,79 @@
+# Copyright 2020 The Actions Ecosystem Authors
+# Modifications Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Portions of this file are derived from the README examples in the Action
+# Slack Notifier project. The original source code was retrieved on
+# January 5, 2022 from:
+
+#     https://github.com/actions-ecosystem/action-slack-notifier/blob/fc778468d09c43a6f4d1b8cccaca59766656996a/README.md
+
+name: Slack Issue Notifications
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check membership
+        id: is_organization_member
+        uses: jamessingleton/is-organization-member@v1
```

Comment:
Can you switch this to use the commit SHA instead of floating tags? Safer.

Reply:
(And in the original workflow. Sorry for missing it the first time around!)

Reply:
Using [fb5924a6287762ee5fc71bf9e95a60842af5528d](https://github.com/JamesSingleton/is-organization-member/tree/fb5924a6287762ee5fc71bf9e95a60842af5528d) (v1.0.1) here.